util/configure

```diff
@@ -381,10 +385,9 @@ if ($platform eq 'msys') {
 
 print "platform: $platform ($OS)\n";
 
-my $ngx_prefix;
 if ($platform eq 'msys') {
     $ngx_prefix = "$prefix";
-} else {
+} elsif (!$ngx_prefix) {
```

Comment:
It' s weird: can't `msys` platform specify `$nginx_prefix` using the command line option?

Reply:
updated